assets/src/styles/blocks/PageHeader.scss

```diff
@@ -62,9 +62,9 @@ $text-column-padding-in: 24px;
     padding: 0;
     font-size: inherit;
     font-weight: inherit;
-    line-height: var(--page-header-title--line-height, 1em);
```

Comment:
The `--page-header-title--line-height` is not being used in our code. It seemed to be used by campaigns a while ago.